.travis.yml

```diff
@@ -8,9 +8,11 @@ before_install:
   - scripts/travis_es.sh
   - /tmp/elasticsearch/elasticsearch-1.3.7/bin/elasticsearch -d -D es.path.data=/tmp -D es.gateway.type=none -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false
 install:
+  - pip install flake8
   - pip install --no-deps --exists-action=w --download-cache=/tmp/pip-cache -r requirements/test.txt --find-links https://pyrepo.addons.mozilla.org/wheelhouse/
 before_script:
   - mysql -e 'create database zamboni;'
+  - flake8 . --exclude=./docs/conf.py
```

Comment:
Chatted to rob and we thought that since its all working with these rules in, we'd leave them in. In the future we won't ignore these rules.